src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothLeAdapter.cs

```diff
@@ -269,19 +270,31 @@ public BluetoothLePacketType PacketType
         /// <exception cref="NotSupportedException">Thrown when the Bluetooth LE is not supported.</exception>
         /// <exception cref="InvalidOperationException">Thrown when the Bluetooth LE is not enabled.</exception>
         /// <since_tizen> 3 </since_tizen>
+        [Obsolete("Deprecated since API level 6. Please use GetServiceUuid() method on BluetoothLeDevice.")]
         public IEnumerable<string> ServiceUuid
         {
             get
             {
-                if (BluetoothAdapter.IsBluetoothEnabled && Globals.IsInitialize)
-                {
-                    Log.Info(Globals.LogTag, "Retrieving Service uuid- ");
-                    return BluetoothLeImplAdapter.Instance.GetLeScanResultServiceUuids(_scanData, _packetType);
-                }
-                return null;
+                return GetServiceUuid(_packetType);
             }
         }
 
+        /// <summary>
+        /// Gets the service UUIDs list from the LE scan result information.
+        /// </summary>
+        /// <value> Gets the list of the string service UUIDs.</value>
+        /// <remarks>The Bluetooth must be enabled.</remarks>
+        /// <param name="packetType"> The enumeration for BLE packet type.</param>
+        /// <returns>The service uuid value</returns>
+        /// <feature>http://tizen.org/feature/network.bluetooth.le</feature>
+        /// <exception cref="NotSupportedException">Thrown when the Bluetooth LE is not supported.</exception>
+        /// <exception cref="InvalidOperationException">Thrown when the Bluetooth LE is not enabled.</exception>
+        /// <since_tizen> 6 </since_tizen>
+        public IEnumerable<string> GetServiceUuid(BluetoothLePacketType packetType)
+        {
+            return BluetoothLeImplAdapter.Instance.GetLeScanResultServiceUuids(_scanData, packetType);
+        }
+
         /// <summary>
         /// Gets the device name from the LE scan result information.
```

Comment:
I think scan result is information, so I don't think information is not needed in the sentence.